misago/threads/tests/test_thread_bulkpatch_api.py

```diff
@@ -79,8 +79,8 @@ def test_invalid_input_keys(self):
         self.assertEqual(
             response.json(),
             {
-                "ids": ["A valid integer is required."],
-                "ops": ['Expected a dictionary of items but got type "int".'],
+                "ids": {"0": ["A valid integer is required."]},
```

Comment:
Changed response shape will break the UI.

Reply:
I didn't change response. Maybe something changed in rest_framework? Look on def _get_error_details in venv/misago/lib/python3.6/site-packages/rest_framework/exceptions.py It's look like depend on input data type.